app/structure/navBar/navBarController.js

```diff
@@ -1,11 +1,18 @@
 'use strict';
 
 define([''], function() {
-  return function($rootScope, authenticationService, AUTH_EVENTS) {
+  return function($state, $rootScope, authenticationService, AUTH_EVENTS) {
 
     this.logOut = function() {
       authenticationService.logOut();
     };
 
+    this.classFor = function(link) {
+      if ($state.current.name.startsWith('app.' + link)) {
```

Comment:
I'm about to merge in my work with the loading spinners. It contains a service that you may want to add to, pulling this out of the navBarController. We can chat when it's merged.